pkg/udev/init.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+RULESDIR="/run/udev/rules.d"
+
+echo "Starting udev daemon..."
+mkdir -p "$RULESDIR"
+
+# Copy existent rules in hostfs
+find /hostfs/lib/udev/ -maxdepth 1 -type f -exec cp {} /lib/udev/ \;
+find /hostfs/lib/udev/rules.d/ -maxdepth 1 -type f -exec cp {} /lib/udev/rules.d/ \;
+
+# Start daemon
+udevd --daemon
+udevadm hwdb --update
+udevadm trigger
+sleep inf
```

Comment:
Can this be an onboot container? instead of a service container Or does the daemon process not fit in with that?

Reply:
I think it fits better as a service container, onboot containers are intended to run one shot settings and exit. Also, by using service container it makes more user friendly to enter in the container, run udev monitor, see errors, even restart the container if needed....

Reply:
Thinking more about it since you mentioned that, perhaps udevd could fit as an init container, like udevd, so neither onboot nor service container, I will make some test and update this PR....